### DISCUSSION TITLE
Cant Install Uno.SDK

### DISCUSSION THREAD

**Discussion by xDreamms** [unoplatform/uno#18282] [Q&A]
### Current behavior

» Attempting to fix: Uno SDK
SHELL: dotnet restore
C:\Users\X\AppData\Local\Temp\Uno.Sdk-4041af57-9c16-46f8-9cab-5cb560ac24d3\Uno. Sdk.csproj : error : SDK "Uno.Sdk" could not be resolved. Exactly one of the following probe messages indicates why we were unable to resolve the SDK. Examine and resolve this message to correctly specify the SDK. C:\Users\X\AppData\Local\Temp\Uno.Sdk-4041af57-9c16-46f8-9cab-5cb560ac24d3\Uno.Sdk.csproj : error : SDK resolver "Microsoft.DotNet.MSBuildWorkloadSdkResolver" returned null.
C:\ Users\X\AppData\Local\Temp\Uno.Sdk-4041af57-9c16-46f8-9cab-5cb560ac24d3\Uno.Sdk.csproj : error : Uno.Sdk package cannot be found. There is no package with this id in sources: Microsoft Visual Studio Offline Packages
C:\Users\X\AppData\Local\Temp\Uno.Sdk-4041af57-9c16-46f8-9cab-5cb560ac24d3\Uno.Sdk.csproj : error MSB4236: Specified ' SDK 'Uno.Sdk/5.3.108' could not be found.

### Expected behavior

Should install uno.sdk successfully

**Comment by jeromelaban** [comment]
Thanks for the report. Could you give a try to https://platform.uno/docs/articles/common-issues-all-ides.html#could-not-resolve-sdk-unosdk ?
